pkg/loop/cmd/loopinstall/install.go

```diff
@@ -123,6 +123,17 @@ func downloadAndInstallPlugin(pluginType string, pluginIdx int, plugin PluginDef
 		}
 	}
 
+	// Build env vars from defaults, environment variable, and plugin-specific settings
+	envVars := defaults.EnvVars
+	if envEnvVars := os.Getenv("CL_PLUGIN_ENVVARS"); envEnvVars != "" {
+		envVars = strings.Fields(envEnvVars)
+	}
+
+	// Append plugin-specific env vars
+	if len(plugin.EnvVars) != 0 {
+		envVars = append(envVars, plugin.EnvVars...)
+	}
```

Comment:
Could there be duplicate env vars that are plugin specific? If so we should validate or merge and maybe let the plugin env var override any defaults or those specified from `CL_PLUGIN_ENVVARS`?

Reply:
Good idea! We want the following precedence:
- defaults
- `CL_PLUGIN_ENVVARS`
- plugins

Plus either replacing existing ones or appending new.